src/test/java/org/folio/innreach/controller/d2ir/InnReachCirculationControllerTest.java

```diff
@@ -291,4 +294,79 @@ void precessReportUnshippedItemReceived_whenTransactionItemShipped() {
     assertNotEquals(RECEIVE_UNANNOUNCED, transactionUpdated.getState());
   }
 
+  @Test
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/inn-reach-transaction/pre-populate-inn-reach-transaction.sql"
+  })
+  void checkTransactionIsInStateItemReceived() {
```

Comment:
checkTransactionIsInStateItemReceived() and checkTransactionIsInStateReceiveUnannounced() methods perform the same steps and validation apart from setting the initial state. They can be combined into a single test with different inputs: 
- intial state = ITEM_RECEIVED
- intial state = RECEIVE_UNANNOUNCED

You can use @ParametrizedTest to achieve the above. A couple of samples:
- https://github.com/folio-org/mod-inn-reach/blob/4497d0f40df3183b7a108e362f1787274b3933de/src/test/java/org/folio/innreach/config/props/FolioEnvironmentTest.java#L42
- https://github.com/folio-org/mod-inn-reach/blob/4497d0f40df3183b7a108e362f1787274b3933de/src/test/java/org/folio/innreach/controller/d2ir/TransferRequestCirculationApiTest.java#L72

Reply:
fixed